### ISSUE TITLE
Error if fails to find config file

### ISSUE THREAD

**Issue by PerditionC** [FDOS/fdisk#3]
Lastest version has cryptic error message when fails to find optional fdisk.ini configuration file.

**Comment by micahcochran** [comment]
This line is in FDISKIO.C:
```C
printf("Error encountered on line %d of the \"fdisk.ini\" file...Program Terminated.\n",line_counter);
```
That exact error message is in the source file 26 times.

A couple look like there is a line that exceeds 254 characters.  (Quickly looking at the code.)

```C
printf("Error encountered on line %d of the \"fdisk.ini\" file.  Setting \"%s\" has a  value of \"%s\"...Program Terminated.\n",line_counter, command_buffer, setting_buffer);
```

An example of how this might work:
```
FDISK.INI
; comments for 9 lines
        REBOOT        InVaLiD
; more stuff

CONSOLE Error Message
Error encountered on line 10 of the "fdisk.ini" file.  Setting "REBOOT" has a  value of "InVaLiD"...Program Terminated.
```

**Comment by micahcochran** [comment]
Sorry,  I went off on a tangent there.  That's really a separate issue within processing FDISK.INI file.

So, to test this I renamed FDISK.INI to FDISK_BK.INI.

The problem occurs in FDISK 1.3.1.

![Screenshot from 2021-01-23 06-26-55](https://user-images.githubusercontent.com/7433104/105578253-5929c100-5d44-11eb-82f8-c6fcccb3443a.png)

In FDISK 1.2.1, it doesn't care about the FDISK.INI.  In `Process_Fdiskini_File(`), it appears to just use a default configuration.  That's bad behavior.

A more preferable way to behave is report to the user by printing "Unable to find \"fdisk,ini\" configuration file."
Prompt the user "Use the default configuration (Y/N). [N]?"
Yes : use the default configuration and continue.
No: exit

That way the user gets to decide if using the default FDISK configuration is problematic or not.
